crates/vsock/README.md

```diff
@@ -43,6 +43,16 @@ Run the vhost-user-vsock device:
 vhost-user-vsock --guest-cid=<CID assigned to the guest> \
   --socket=<path to the Unix socket to be created to communicate with the VMM via the vhost-user protocol>
   --uds-path=<path to the Unix socket to communicate with the guest via the virtio-vsock device>
+  -f=<path to the local yaml configuration file>
+```
+
+Configuration Example
+
+```yaml
+vhost:
```

Comment:
What `vhost` means here?

Reply:
I originally thought  about spliting host part from the sibling part.  Was that sensible? Now I switched to your suggested

``` yaml
vms:
...
```